src/test/java/posters/pageobjects/pages/checkout/PaymentPage.java

```diff
@@ -66,27 +66,27 @@ public PlaceOrderPage selectCreditCard(int position)
         // Clicks the continue button
         $("#btnUsePayment").scrollTo().click();
 
-        return new PlaceOrderPage();
+        return new PlaceOrderPage().isExpectedPage();
     }
 
     @Step("fill and send payment form")
-    public PlaceOrderPage sendPaymentForm(String number, String name, String month, String year)
+    public PlaceOrderPage sendPaymentForm(CreditCard card)
     {
         // Credit Card Number
         // Fills the card number field with the parameter
-        creditCardNumber.val(number);
+        creditCardNumber.val(card.getCardNumber());
```

Comment:
@oomelianchuk Could we create a credit card form and reuse it at the possible places if the underlying form is the same?

Reply:
Done